src/fuzz_introspector/analyses/function_call_analyser.py

```diff
@@ -0,0 +1,260 @@
+        html_string += (
+            "<p>"
+            "This section shows a list of 3rd party function "
+            "calls and their relative coverage information. "
+            "By static analysis of the target project code, "
+            "all of the 3rd party function call and their caller "
+            "information, including the source file and line "
+            "number that initiate the call are captured. "
+            "The caller source code file and line number are "
+            "shown in column 2 while column 1 is the function "
+            "name of the 3rd party function call. Each occurrent "
+            "of the 3rd party function call will occuply "
+            "a separate row. Column 3 of each row indicate if "
+            "the 3rd party call in the source file line is "
+            "unreachable. Column 4 lists all fuzzers that have "
+            "covered that particular system call in "
+            "that specific location (source file and line)"
+            "during their dynamic fuzzing."
+            "</p>"
+        )
+
+        html_string += html_helpers.html_add_header_with_link(
+            "Function in each files in report",
+            2,
+            toc_list
+        )
+
+        # Third party function calls table
+        tables.append(f"myTable{len(tables)}")
+        html_string += html_helpers.html_create_table_head(
+            tables[-1],
+            [
+                ("Function name", ""),
+                ("Source file",
+                 "Source file, line number and parent function of this function call. "
+                 "Based on static analysis."),
+                ("Reachable code? ",
+                 "Is this code reachable by any functions? "
+                 "Based on static analysis."),
+                ("Reached by Fuzzers",
+                 "The specific fuzzers that reach this function call. "
+                 "Based on static analysis.")
+            ]
+        )
+
+        for fd in func_profile_list:
+            func_name = utils.demangle_cpp_func(fd.function_name)
+
+            if fd.function_name in called_func_dict.keys():
+                called_func_list = called_func_dict[fd.function_name]
+                if len(called_func_list) == 0:
+                    called_func_list = [""]
+            else:
+                called_func_list = [""]
+            for called_func in called_func_list:
+                hit = "Yes" if (called_func in reachable_func_list) else "No"
+
+                fuzzer_hit = False
+                coverage = proj_profile.runtime_coverage
+                for parent_func in fd.incoming_references:
+                    if coverage.is_func_lineno_hit(
+                        parent_func,
+                        int(called_func.split(":")[1])
+                    ):
+                        fuzzer_hit = True
+                        break
+                fuzzer = fd.reached_by_fuzzers if fuzzer_hit else [""]
```

Comment:
Could we name this something more description than `fuzzer`?

Reply:
changed to list_of_fuzzer